pkg/testers/ginkgo/ginkgo.go

```diff
@@ -94,9 +96,43 @@ func (t *Tester) pretestSetup() error {
 	}
 	log.Printf("Using kubeconfig at %s", t.kubeconfigPath)
 
+	if err := t.AcquireTestPackage(); err != nil {
+		return fmt.Errorf("failed to get ginkgo test package from published releases: %s", err)
+	}
+
 	return nil
 }
 
+// TODO(michaelmdresser): change behavior if a local built e2e.test package
+// is available
+func (t *Tester) AcquireTestPackage() error {
+	releaseTar := fmt.Sprintf("kubernetes-test-%s-%s.tar.gz", runtime.GOOS, runtime.GOARCH)
+
+	script := []string{
+		"set -o xtrace",
+		"set -o pipefail",
+		"set -o errexit",
+		"set -o nounset",
+		"cd $ARTIFACTS",
```

Comment:
we shouldn't put this under artifacts, maybe a tempdir?

Reply:
Hmm. We do put other artifacts in here, I think its a reasonable place to download to. Its the only space "owned" by kubetest2 and would let a test runner potentially inspect which version of the test package was downloaded because it will be at a known path.

Reply:
this will end up reuploading the entire kubernetes release tarballs in every job to gcs.
maybe the e2e.test might make sense under artifacts, but the tars should go in a tempdir which we then cleanup.

Reply:
Ah, didn't realize that. Added logic to download to a tempdir and then remove the dir after extracting the test package.